src/layers/spaces.js

```diff
@@ -0,0 +1,100 @@
+import FeatureLayer from '@arcgis/core/layers/FeatureLayer';
+import accessibleIcon from '@fortawesome/fontawesome-free/svgs/solid/wheelchair-move.svg';
+import chargingStationIcon from '@fortawesome/fontawesome-free/svgs/solid/charging-station.svg';
+import circleDollarToSlotIcon from '@fortawesome/fontawesome-free/svgs/solid/circle-dollar-to-slot.svg';
+import circleXmarkIcon from '@fortawesome/fontawesome-free/svgs/solid/circle-xmark.svg';
+import motorcycleIcon from '@fortawesome/fontawesome-free/svgs/solid/motorcycle.svg';
+import peopleCarryBoxIcon from '@fortawesome/fontawesome-free/svgs/solid/people-carry-box.svg';
+import peopleGroupIcon from '@fortawesome/fontawesome-free/svgs/solid/people-group.svg';
+import personWalkingLuggageIcon from '@fortawesome/fontawesome-free/svgs/solid/person-walking-luggage.svg';
+import squareParkingIcon from '@fortawesome/fontawesome-free/svgs/solid/square-parking.svg';
+import truckRampBoxIcon from '@fortawesome/fontawesome-free/svgs/solid/truck-ramp-box.svg';
+import massachusettsIcon from '../assets/massachusetts.svg';
```

Comment:
These are probably going to just be placeholders for now.

Later on in the roadmap (if we get that far) I'd be open to fleshing these out a bit more, but I don't care to do that currently as I'm just polishing/pushing up a bunch of my "exploratory" work for now.